src/components/button/Button.tsx

```diff
@@ -35,9 +69,11 @@ export function Button(props: ButtonProps) {
     disabled = !tooltipProps.content,
     ...otherToolTipProps
   } = tooltipProps;
+
   const InnerButton = buttonProps.disabled
-    ? BlueprintAnchorButton
-    : BlueprintButton;
+    ? styled(BlueprintAnchorButton)(css(children))
+    : styled(BlueprintButton)(css(children));
```

Comment:
Never called `styled()` inside a component. This creates a new component which makes React rerender the whole subtree every time.